lib/internal/Magento/Framework/App/Response/Http.php

```diff
@@ -131,7 +131,7 @@ public function sendVary()
      */
     public function setPublicHeaders($ttl)
     {
-        if ($ttl < 0 || !preg_match('/^[0-9]+$/', $ttl)) {
+        if ($ttl === null || $ttl < 0 || !preg_match('/^[0-9]+$/', $ttl)) {
```

Comment:
Hello @anzin 
Perhaps we need to fix tests, but not code.
The `ttl` param is `integer` and not null
What do you think about this?

Reply:
Hello @andrewbess, thanks for your review, I check test and it created to check this method without parameter. I can revert my change `$ttl === null` and remove test,  but it seems to me that it is better to leave check of NULL. Since the method is public and does not have strict parameter typing.

What you think?

Thank you.